aiohasupervisor/models/os.py

```diff
@@ -124,3 +124,19 @@ class YellowOptions(Options):
     disk_led: bool | None = None
     heartbeat_led: bool | None = None
     power_led: bool | None = None
+
+
+@dataclass(frozen=True, slots=True)
+class SwapInfo(Options):
```

Comment:
```suggestion
class SwapInfo(ResponseData):
```
`Options` is only for models of objects used in request post bodies, `ResponseData` is for models of response objects